bark/runtime/viewer/viewer.py

```diff
@@ -481,9 +489,26 @@ def drawRssSafetyResponses(self, world, ego_id):
             self.drawPolygon2d(transformed_polygon, response_color,
                                0.6, response_color, zorder=9)
 
+    def DrawRSSBrakingDistances(self, world, eval_id):
```

Comment:
does this also work when using the EvaluatorRSS and not the BehaviorRSSCompliant? Maybe add a docstring to indicate usage?

Reply:
Yes, it always works. Thus, it is nor required in my opinion